File: peek_core_device/plugin-module/_private/tuples/admin/BuildUpdateAction.ts
import {addTupleType, TupleActionABC} from "@synerty/vortexjs";
import {deviceTuplePrefix} from "../../PluginNames";


@addTupleType
export class BuildUpdateAction extends TupleActionABC {
    public static readonly tupleName = deviceTuplePrefix + "BuildUpdateAction";

    deviceType: string;

    constructor() {
        super(BuildUpdateAction.tupleName)
    }
}

File: peek_core_device/_private/tuples/BuildUpdateAction.py
from vortex.Tuple import Tuple, addTupleType, TupleField

from peek_core_device._private.PluginNames import deviceTuplePrefix


@addTupleType
class BuildUpdateAction(Tuple):
    """ Build Update Action

    This instructs the

    """
    __tupleType__ = deviceTuplePrefix + 'BuildUpdateAction'

    #:  Description of date1
    deviceType:str = TupleField()